interp/frame.go

```diff
@@ -244,13 +244,18 @@ func (fr *frame) evalBasicBlock(bb, incoming llvm.BasicBlock, indent string) (re
 			case callee.Name() == "runtime.hashmapStringSet":
 				// set a string key in the map
 				m := fr.getLocal(inst.Operand(0)).(*MapValue)
+				// "key" is a single LLVM value in emitMapUpdate, but in the frame it is
+				// represented by a (pointer, len) pair
```

Comment:
Perhaps this comment would be clearer when you describe why this happens:

> "key" is a Go string value, which in the TinyGo calling convention is split up in separate pointer and length parameters.

See for example the declaration of the called function (here: https://github.com/aykevl/tinygo/blob/master/src/runtime/hashmap.go#L273):

```go
func hashmapStringSet(m *hashmap, key string, value unsafe.Pointer)
```

Reply:
Added the comment to https://github.com/aykevl/tinygo/pull/160